level-ws.js

```diff
@@ -124,15 +112,14 @@ WriteStream.prototype._flush = function (f) {
     }
   }
 }
-
 WriteStream.prototype.toString = function () {
   return 'LevelUP.WriteStream'
 }
 
 WriteStream.prototype.destroy = function () {
```

Comment:
If we define `_destroy` instead of `destroy` then we don't need to manage `destroyed` state anymore. Also, do we still need `this.writable`?